package.json

```diff
@@ -45,6 +46,7 @@
     "babel-preset-react-native": "2.0.0",
     "eslint": "^3.19.0",
     "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-angular": "^3.0.0",
```

Comment:
when i run npm run lint, it will complain and break eslint-plugin-angular is not installed.  I will remove it.